index.js

```diff
@@ -56,6 +56,25 @@ module.exports = {
     'import/no-named-default': 'error',
     'import/no-webpack-loader-syntax': 'error',
     'import/prefer-default-export': 'error',
+    'jsdoc/check-examples': 0,
+    'jsdoc/check-param-names': 1,
```

Comment:
To maintain file convention we should rather write `'error'` and `'off`' instead of `1` and `0`

Reply:
Sure. `1` equals `warn`. Are you also suggesting that `error` should be used?
Maybe it would a better idea.

Reply:
`warn` will be just ignored, we need CI crash to enforce it's respected.

Reply:
fixed